src/components/sections/items/TextInputSectionItem.tsx

```diff
@@ -102,40 +115,56 @@ export const TextInputSectionItem = forwardRef<InternalTextInput, TextProps>(
           inlineLabel ? styles.containerInline : styles.containerMultiline,
           topContainerStyle,
           containerPadding,
-          borderColor,
+          getBorderColor(),
         ]}
         onAccessibilityEscape={accessibilityEscapeKeyboard}
       >
         <ThemeText type="body__secondary" style={styles.label}>
           {label}
         </ThemeText>
-        <InternalTextInput
-          ref={combinedRef}
-          style={[
-            styles.input,
-            inlineLabel ? contentContainer : undefined,
-            padding,
-            style,
-          ]}
-          placeholderTextColor={theme.text.colors.secondary}
-          onFocus={onFocusEvent}
-          onBlur={onBlurEvent}
-          maxFontSizeMultiplier={MAX_FONT_SCALE}
-          {...props}
-        />
-        {showClear ? (
-          <View style={styles.inputClear}>
-            <PressableOpacity
-              accessible={true}
-              accessibilityRole="button"
-              accessibilityLabel={t(SectionTexts.textInput.clear)}
-              hitSlop={insets.all(8)}
-              onPress={onClearEvent}
-            >
-              <ThemeIcon svg={Close} />
-            </PressableOpacity>
+        <View style={styles.inputContainer}>
+          <InternalTextInput
+            ref={combinedRef}
+            style={[
+              styles.input,
+              inlineLabel ? contentContainer : undefined,
+              padding,
+              style,
+            ]}
+            placeholderTextColor={theme.text.colors.secondary}
+            onFocus={onFocusEvent}
+            onBlur={onBlurEvent}
+            maxFontSizeMultiplier={MAX_FONT_SCALE}
+            {...props}
+          />
+          {showClear ? (
+            <View style={styles.inputClear}>
+              <PressableOpacity
+                accessible={true}
+                accessibilityRole="button"
+                accessibilityLabel={t(SectionTexts.textInput.clear)}
+                hitSlop={insets.all(8)}
+                onPress={onClearEvent}
+              >
+                <ThemeIcon svg={Close} />
+              </PressableOpacity>
+            </View>
+          ) : null}
+        </View>
+        {errorText !== undefined && (
+          <View
+            style={styles.error}
+            accessibilityLiveRegion="polite"
```

Comment:
Nice solution! I took a look at some aria doc to see if a specific role also would be appropriate, and think maybe role "alert" would be nice here. Some cliffs from the doc:
> The alert role is used to communicate an important and usually time-sensitive message to the user. When this role is added to an element, the browser will send out an accessible alert event to assistive technology products which can then notify the user.
>
> The alert role should only be used for information that requires the user's immediate attention, for example:
>
>- An invalid value was entered into a form field

Maybe check if when adding accessibility role `alert`, is the announcements done even without adding live region?

Reply:
Added `role: 'alert'` now. Does however look like accessibilityLiveRegion is only supported by Android, so exchanged it with focusRef that in turn will trigger the screen reader to read out error message when it appears for both android and ios.